core/ulog/log_test.go

```diff
@@ -134,6 +134,24 @@ func TestConfiguredLoggerWithTextFormatter(t *testing.T) {
 	})
 }
 
+func TestConfiguredLoggerWithTextFormatter_NonDev(t *testing.T) {
+	dev := development
+	development = false
+	defer func() {
+		development = dev
+	}()
+	withLogger(t, func(tmpDir string, logFile string) {
+		log := Logger()
+		txt := true
+		log.Configure(Configuration{
+			Level:         "debug",
+			TextFormatter: &txt,
+		})
+		zaplogger := log.RawLogger()
+		assert.Equal(t, zap.DebugLevel, zaplogger.Level())
```

Comment:
this makes me sort of nervous. i'm thinking we need to have an in-built set of defaults (e.g. YAML byte slices) that contain the configuration defaults per-environment and then we can load those explicitly. the more checks i see in our tests and production code `if environment == ...` the more brittle our software becomes. perhaps we can talk in person about this?